conda-forge.yml

```
@@ -20,4 +20,5 @@ github_actions:
   - pull_request
 provider:
   linux_64: github_actions
+  linux_aarch64: default
```

Remove the "timeouts" at the top,
```suggestion
  linux_aarch64: azure
```

Add in the `build_platform` section, set it to:
```
build_platform:
    # Cross compilation
    linux_aarch64: linux_64
    # no cross compilation (through emulation...., super slow)
    linux_aarch64: linux_64
```

depending on if you want cross compilaton or not

---

Please skip all of OSX and skip all but one version of python for now as well. The python build matrix is pretty low risk. 

Let’s do our part to save 4kWhr of power for the planet.

---

How do we skip pythons - setting it explicitly in the build dependencies? (ge python =3.10 everywhere?)